docs/source/user_guide/data_check_actions.ipynb

```diff
@@ -301,47 +292,6 @@
     "results_no_errors = search_iterative(X_train_no_errors, y_train_no_errors, problem_type='binary')\n",
     "results_no_errors"
    ]
-  },
```

Comment:
After these changes, the only error left is that the target has null. That leaves little reason to keep this in, but I think a separate issue should be filed to create a more robust example using more data checks with errors.

Reply:
Sure, please feel free to file and complete!